![Image description](https://github.com/yuchaojiang/BIOSBCB785/blob/master/Title.png)

### Instructor

[Yuchao Jiang](https://yuchaojiang.github.io/), Assistant Professor, Departments of Biostatistics & Genetics, UNC Chapel Hill<br /> 
Office: 4115D McGavran-Greenberg Hall<br /> 
Phone:  919-843-3656<br /> 
Email:  yuchaoj@email.unc.edu (contact via slack is preferred)


### Course Information

* **Description**: This course is designed to provide graduate students interested in statistical genetics and genomics with an opportunity to gain or enhance knowledge in gene expression analysis by next-generation sequencing. The course is aimed at preparing students for conducting methodological research in high-throughput transcriptomic studies. The course includes two sections: bulk RNA-seq and single-cell RNA-seq (scRNA-seq). Each section starts with biological background, followed by statistical and computational methods, and finishes with biological interpretations and follow-ups. Topics include data normalization, measurement of error models, dispersion shrinkage, dimensionality reduction, zero-inflated factor analysis, batch correction, clustering algorithm, deconvolution, pseudotime reconstruction, deep neural network, and autoencoder, etc.

* **[Course Slack](http://unc785spring2023.slack.com/)**

* **[Syllabus](https://www.dropbox.com/s/wjb3c4ihva60f3g/BIOS785_Spring2023_Yuchao_Jiang.pdf?dl=0)**

* **[Course Schedule](https://docs.google.com/spreadsheets/d/1TjQCCgqk-1s0F8PrYYB7yeJAWmy6xEWw5RiEmpNhVRw/edit?usp=sharing)**

* **Class Time & Location**: Tuesdays and Thursdays, 9:30am â€“ 10:45am; McGavran-Greenberg 2301.

* **Office Hours**: Thursdays 11am - noon; McGavran-Greenberg 4115D.

* **Grader**: Wancen Mu (wancen@live.unc.edu)


### Lecture Slides/Papers/Videos

* **Lecture 1**: Introduction to Molecular Biology ([slides](https://www.dropbox.com/s/8gmtr40ru33od5n/Lecture_1.pdf?dl=0))
* **Lecture 2**: Introduction to RNA Sequencing ([slides](https://www.dropbox.com/s/iq049r01h5hy496/Lecture_2.pdf?dl=0), [paper](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0881-8))
* **Lecture 3**: RNA Sequencing Data Normalization ([slides](https://www.dropbox.com/s/q3buwqo7b29b8vl/Lecture_3.pdf?dl=0), [paper](https://www.nature.com/articles/nbt.2931))
* **Lecture 4**: RNA Sequencing Batch Correction ([slides](https://www.dropbox.com/s/hpe3y5j7ssfy5kw/Lecture_4.pdf?dl=0), [paper1](https://academic.oup.com/biostatistics/article/8/1/118/252073), [paper2](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.0030161))
* **Lecture 5**: Isoform Expression ([slides](https://www.dropbox.com/s/yc32x8roea1uk9i/Lecture_5.pdf?dl=0), [paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2666817/))
* **Lecture 6**: Allele-Specific Gene Expression and Mapping of eQTLs ([slides](https://www.dropbox.com/s/m7a2x1ffxwb287i/Lecture_6.pdf?dl=0), [paper](https://onlinelibrary.wiley.com/doi/abs/10.1111/j.1541-0420.2011.01654.x))
* **Lecture 7**: Bulk Gene Expression Deconvolution ([slides](https://www.dropbox.com/s/xfxtgxnczgdo22d/Lecture_7.pdf?dl=0), [paper1](https://www.nature.com/articles/nmeth.1439), [paper2](https://www.nature.com/articles/nmeth.3337))
* **Lecture 8**: Differential Gene Expression Analysis by Bulk RNA Sequencing ([slides](https://www.dropbox.com/s/0yctgxsa4gp0nim/Lecture_8.pdf?dl=0), [paper](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-014-0550-8))
* **Lecture 9**: Gene Set Enrichment Analysis ([slides](https://www.dropbox.com/s/80n38aj9ws3h7b3/lecture_9_new.pdf?dl=0), [paper1](https://www.pnas.org/content/102/43/15545.long), [paper2](https://academic.oup.com/nar/article/40/17/e133/2411151))
* **Lecture 10**: Multiple Hypothesis Testing Correction ([slides](https://www.dropbox.com/s/dy1geo1qly8skhv/Lecture_10.pdf?dl=0), [paper](https://www.pnas.org/content/100/16/9440.long))
* **Lecture 11**: Introduction to Single-Cell RNA Sequencing ([slides](https://www.dropbox.com/s/kafci5whovtj614/Lecture_11.pdf?dl=0), [paper1](https://www.sciencedirect.com/science/article/pii/S1097276515002610), [paper2](https://www.nature.com/articles/nrg3833))

### Tutorials & Workflows

* [RNA-seq analysis](https://github.com/crazyhottommy/RNA-seq-analysis)
* [RNA-seqlopedia](https://rnaseq.uoregon.edu)
* [Informatics for RNA-seq](https://github.com/griffithlab/rnaseq_tutorial)
* [Analysis of single cell RNA-seq data](https://hemberg-lab.github.io/scRNA.seq.course)
* [Awesome single cell](https://github.com/seandavi/awesome-single-cell)
* [simpleSingleCell](http://bioconductor.org/packages/simpleSingleCell)
* [Orchestrating Single-Cell Analysis with Bioconductor](https://osca.bioconductor.org)


### Resources & Datasets

* [GTEx Portal](https://gtexportal.org)
* [10X Genomics single-cell expression](https://support.10xgenomics.com/single-cell-gene-expression/datasets)
* [Single Cell Portal](https://portals.broadinstitute.org/single_cell)
* [Single Cell Expression Atlas](https://www.ebi.ac.uk/gxa/sc)
* [conquer](http://imlspenticton.uzh.ch:3838/conquer)
* [Human Cell Atlas](https://www.humancellatlas.org/)
* [Mouse Cell Atlas](http://bis.zju.edu.cn/MCA)


### Other Resources

* [Tips for Presentation](https://www.dropbox.com/s/k5ymqz8qflpeskl/Tips_for_presentations.pdf?dl=0)
* [Guidelines for Written Review](https://www.dropbox.com/s/rr82bz5alp8ewcx/Written_review_assignments.pdf?dl=0)
