![Image description](https://github.com/yuchaojiang/BIOSBCB785/blob/master/Title.png)

### Instructor

[Yuchao Jiang](https://yuchaojiang.github.io/), Assistant Professor, Departments of Biostatistics & Genetics, UNC Chapel Hill<br /> 
Office: 4115D McGavran-Greenberg Hall<br /> 
Phone:  919-843-3656<br /> 
Email:  yuchaoj@email.unc.edu (contact via slack is preferred)


### Course Information

* **Description**: This course is designed to provide graduate students interested in statistical genetics and genomics with an opportunity to gain or enhance knowledge in gene expression analysis by next-generation sequencing. The course is aimed at preparing students for conducting methodological research in high-throughput transcriptomic studies. The course includes two sections: bulk RNA-seq and single-cell RNA-seq (scRNA-seq). Each section starts with biological background, followed by statistical and computational methods, and finishes with biological interpretations and follow-ups. Topics include data normalization, measurement of error models, dispersion shrinkage, dimensionality reduction, zero-inflated factor analysis, batch correction, clustering algorithm, deconvolution, pseudotime reconstruction, deep neural network, and autoencoder, etc.

* **[Course Slack](http://unc785fall2020.slack.com/)**

* **[Syllabus](https://www.dropbox.com/s/z5tqdqnqdtrma5b/BIOS785_Fall2020_Yuchao_Jiang.pdf?dl=0)**

* **[Schedule](https://docs.google.com/spreadsheets/d/1ohj0Wa0zRMdSLZfHQUVki6c1hKWVc77YZRZwVgZgXU8/edit?usp=sharing)**

* **Class Time & Location**: Mondays and Wednesdays, 9:20am â€“ 10:35am; remote teaching via Zoom.

* **Office Hours**: virtual via [whereby](https://whereby.com/yuchaoj); time TBD.

* **Grader**: TBD.


### Lecture Slides

* **Lecture 1**: Introduction to Molecular Biology ([slides](https://www.dropbox.com/s/ur7oevlh3y09xb0/Lecture_1.pdf?dl=0), [video](https://www.dropbox.com/s/1sjzh5svf1sp9xh/lecture1.mp4?dl=0))
* **Lecture 2**: Introduction to RNA Sequencing ([slides](https://www.dropbox.com/s/e9d4jifqvn0rwa7/Lecture_2.pdf?dl=0), [paper](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0881-8))


### Tutorials & Workflows

* [RNA-seq analysis](https://github.com/crazyhottommy/RNA-seq-analysis)
* [RNA-seqlopedia](https://rnaseq.uoregon.edu)
* [Informatics for RNA-seq](https://github.com/griffithlab/rnaseq_tutorial)
* [Analysis of single cell RNA-seq data](https://hemberg-lab.github.io/scRNA.seq.course)
* [Awesome single cell](https://github.com/seandavi/awesome-single-cell)
* [simpleSingleCell](http://bioconductor.org/packages/simpleSingleCell)
* [Orchestrating Single-Cell Analysis with Bioconductor](https://osca.bioconductor.org)


### Resources & Datasets

* [GTEx Portal](https://gtexportal.org)
* [10X Genomics single-cell expression](https://support.10xgenomics.com/single-cell-gene-expression/datasets)
* [Single Cell Portal](https://portals.broadinstitute.org/single_cell)
* [Single Cell Expression Atlas](https://www.ebi.ac.uk/gxa/sc)
* [conquer](http://imlspenticton.uzh.ch:3838/conquer)
* [Human Cell Atlas](https://www.humancellatlas.org/)
* [Mouse Cell Atlas](http://bis.zju.edu.cn/MCA)


### Other Resources

* [Tips for Presentation](https://www.dropbox.com/s/k5ymqz8qflpeskl/Tips_for_presentations.pdf?dl=0)
* [Guidelines for Written Review](https://www.dropbox.com/s/8zo65nok6fuhbs9/written_review_guidelines.pdf?dl=0)
